Backport 4.3: Orphaned packages #25067 keichwa (#3329) (#3338) (#3341)
uyuni-project/uyuni-docs: modules/client-configuration/pages/package-management.adoc

. In the {productname} {webui}, navigate to [guimenu]``Systems``, click the client to compare, and navigate to the menu:Systems[Software > Packages > Profiles] subtab.
    To compare with a stored profile, select the profile and click btn:[Compare].
. To compare with another client, select the client name and click btn:[Compare] to see a list of differences between the two clients.

[[package-management-orphaned-packages]]
== Orphaned Packages

Orphaned packages are packages synchronized by {productname} that are not associated with a software channel.
Therefore, orphaned packages are usually not available to {productname} clients and users cannot install such packages without additional effort.

A package can become orphaned as a result of one of the following events:

* A synchronized repository removes a package.
  By default, {productname} 5.0 unlinks such package from the channel, but does not remove the package.

>   In {productname} 4.3, you can enable this behavior by using the [literal]``strict mode`` channel configuration.

Reviewer: I would NOT keep this in the 5.0 version, it only confuses the user. Rather, have ONLY relevant instructions in the documents that are either 5.0 or 4,3.

Author: ok, done!

>   By default, {productname} 5.0 unlinks such package from the channel, but does not remove the package.
>   In {productname} 4.3, you can enable this behavior by using the [literal]``strict mode`` channel configuration.

Author: ```suggestion
  With  [literal]``strict mode`` channel configuration, {productname} unlinks such a package from the channel, but will not remove the package.
```

* A repository supersedes a package with a newer version and thus removes the previous version.
* The reposync process got interrupted (for example, because of an out of space exception) and thus downloaded packages were not associated with a channel.
* Users manually uploaded packages, but did not associate them with any channel.

Orphaned packages cost space in user environment and cannot be easily distributed to clients because they are not associated with a channel.
Orphaned package can make sense with specific workflows like bootstrapping or customer-specific procedures.

You can view orphaned packages in the {webui} by clicking menu:Software[Manage > Packages > view Packages in no channel]

> {productname} periodically runs a Taskomatic job that searches for orphaned packages and modifies the package organization ID to 1.

Reviewer: ```suggestion
{productname} periodically runs a taskomatic job that searches for orphaned packages and modifies the package organization ID to 1.
```

> This means that you can delete orphaned packages only after the Taskomatic job execution.

Reviewer: ```suggestion
This means that you can delete orphaned packages only after the taskomatic job execution.
```

If you encounter an orphaned package that you cannot delete, wait 24 hours and try the deletion again.

// .Procedure: Deleting orphaned packages

> You can deleted orphaned packages:

Reviewer: ```suggestion
Delete orphaned packages:
```

// * With the UI (FIXME is this possible?  How?

> * with commandline tools targeting individual packages.

Reviewer: ```suggestion
* with command line tools targeting individual packages.
```

  For example:
+

----
spacecmd package_remove zypper-1.14.52-150400.1.9.x86_64
----

> * with commandline tools targeting all orphaned packages at once:

Reviewer: ```suggestion
* with command line tools targeting all orphaned packages at once:
```
